github/ci/prow/files/plugins.yaml

```diff
@@ -66,6 +66,10 @@ plugins:
   - approve
   - lgtm
   - owners-label
+  - config-updater
```

Comment:
The config-update is enabled for the project-infra repo. It contains all configurations. You don't have to add this.